packages/shared-lib/package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "@sofie-automation/shared-lib",
+	"version": "1.44.0-in-development",
+	"private": true,
+	"description": "Library for types & values shared by core, workers and gateways",
+	"main": "dist/index.js",
+	"typings": "dist/index.d.ts",
+	"license": "MIT",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/nrkno/tv-automation-server-core.git",
+		"directory": "packages/sharelib"
+	},
+	"bugs": {
+		"url": "https://github.com/nrkno/tv-automation-server-core/issues"
+	},
+	"homepage": "https://github.com/nrkno/tv-automation-server-core/blob/master/packages/sharelib#readme",
+	"scripts": {
+		"build": "rimraf dist && yarn build:main",
+		"build:main": "tsc -p tsconfig.build.json",
+		"lint:raw": "eslint --ext .ts --ext .js --ignore-pattern dist",
+		"lint": "yarn lint:raw .",
+		"unit": "jest",
+		"test": "yarn lint && yarn unit",
+		"watch": "jest --watch",
+		"cov": "jest --coverage; open-cli coverage/lcov-report/index.html",
+		"cov-open": "open-cli coverage/lcov-report/index.html",
+		"release": "yarn reset && yarn test && yarn changelog",
+		"reset": "git clean -dfx && git reset --hard && yarn",
```

Comment:
This was probably in the other libs, but this reset line is odd to have deep in here. Itll affect the whole clone, so should it really be in here?
(Also, Ive never used it, so question its purpose)

Reply:
I agree, it's also present in corelib, and job-worker. I'm leaving it here as well and we'll remove it in an upcoming cleanup.